app/src/main/java/com/example/pricingpal/model/CSVReader.kt

```diff
@@ -16,7 +16,7 @@ import java.util.Scanner
  *  called in MainActivity.kt using the keyword "this" as the context
```

Comment:
This comment block needs to be changed to reflect the change to an object.  The "this" keyword is no longer used.